client/ayon_core/plugins/publish/extract_thumbnail.py

```diff
@@ -348,17 +349,45 @@ def _get_filtered_repres(self, instance):
                 # to be published locally
                 continue
 
-            if "review" not in tags:
-                continue
-
             if not repre.get("files"):
                 self.log.debug((
                     "Representation \"{}\" doesn't have files. Skipping"
                 ).format(repre["name"]))
                 continue
 
-            filtered_repres.append(repre)
-        return filtered_repres
+            has_review_tag = "review" in tags
+            if not has_review_tag and not self._is_valid_images_repre(repre):
+                continue
+
+            if has_review_tag:
+                review_repres.append(repre)
+            else:
+                other_repres.append(repre)
```

Comment:
Why not just do this?
```suggestion
            if has_review_tag:
                review_repres.append(repre)
            elif self._is_valid_images_repre(repre):
                other_repres.append(repre)
```